jwst/cube_build/cube_build_step.py

```diff
@@ -161,12 +161,19 @@ def process(self, input):
             self.pars_input['output_type'] = 'single'
             self.log.info('Cube Type: Single cubes')
             self.pars_input['coord_system'] = 'skyalign'
-            self.interpolation = 'pointcloud'
 
-            # Don't allow anything but msm or emsm weightings
-            if ((self.weighting != 'msm') and (self.weighting != 'emsm')):
-                self.weighting = 'emsm'
+            # Don't allow anything but drizzle, msm, or emsm weightings
+            if ((self.weighting != 'msm') and (self.weighting != 'emsm') and (self.weighting != 'drizzle')):
```

Comment:
Not a big deal, but this line could be made a little cleaner to read using:
```
if self.weighting not in ['msm', 'emsm', 'drizzle']:
```

Reply:
Done